src/Kiota.Builder/Refiners/CommonLanguageRefiner.cs

```diff
@@ -109,9 +109,7 @@ private static bool DoesAnyParentHaveAPropertyWithDefaultValue(CodeClass current
             }
             var propertyOriginalName = (currentProperty.IsNameEscaped ? currentProperty.SerializationName : current.Name)
                                         .ToFirstCharacterLowerCase();
-            var accessorName = (currentProperty.IsNameEscaped ? currentProperty.SerializationName : current.Name)
-                                .CleanupSymbolName()
-                                .ToFirstCharacterUpperCase();
+            var accessorName = propertyOriginalName.CleanupSymbolName().ToFirstCharacterUpperCase();
```

Comment:
can you provide more context on this one please? I feel that this is going to impact properties that are reserved names in the language.

Reply:
ah right, good catch.